src/CodeGen_ARM.cpp

```diff
@@ -112,13 +112,23 @@ CodeGen_ARM::CodeGen_ARM(Target target)
         casts.push_back(p);
 
         // Saturating add
+#if LLVM_VERSION >= 100
+        if (t.is_int()) {
+            p.intrin32 = "llvm.sadd.sat" + t_str;
+            p.intrin64 = "llvm.aarch64.neon.sqadd" + t_str;
```

Comment:
The standard LLVM intrinsic is probably also supported on arm64, and seems better to use if it's available. It's probably only a matter of time before the aarch64 intrinsic is removed and we'll get another breakage.

Reply:
Done, let's try it out and see how it goes